README.md

```diff
@@ -90,6 +90,8 @@
 42. [**Youtube Multi Downloader**](https://youtubemultidownloader.com) - Download a complete playlist on youtube.
 43. [**Hotstar for Free**](http://kattappa.me/newlive?url=LINK_OF_HOTSTAR_VIDEO)- You can watch live match or anything else without having premium account on HotStar
 44. [**draw.io**](https://www.draw.io/) - Draw flowcharts, ER Diagrams easily.
+45. [**Hackerrank**](https://www.hackerrank.com/), [**Codechef**](https://www.codechef.com/) - Competitive Coding
+46. [**reveal.js**](https://revealjs.com/#/) - Make slides wich open on browser
```

Comment:
This won't be probably used by 99% of the viewers. So I don't see much potential adding this.